Marlin/Marlin_main.cpp

```diff
@@ -2489,13 +2492,13 @@ static void homeaxis(AxisEnum axis) {
 
     if (retracting == retracted[active_extruder]) return;
 
-    float old_feedrate = feedrate;
+    float old_feedrate_mm_m = feedrate_mm_m;
 
     set_destination_to_current();
 
     if (retracting) {
 
-      feedrate = retract_feedrate_mm_s * 60;
+      feedrate_mm_m = MMS_TO_MMM(retract_feedrate_mm_s);
```

Comment:
IMO this is an overkill.. if it was a crazy complicated calculation I could understand.

I am not really a macro fanboy ! :laughing: